frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/vm.ts

```diff
@@ -71,16 +70,16 @@ Object.freeze(networkTabCol);
 
 // Storage
 export enum DISK_SOURCE {
```

Comment:
I don't think its a good idea maintaining these `enum` s here...
Every time something changes in `src/` and it isn't updated here the tests break.

I propose another solution - to have these `enum` s declared in `src/` and imported from there:
I created a small PR for the disks as an example of what I think should be done: https://github.com/openshift/console/commit/7d8179cc5fec93e2ad5396ac80ae1949e25024b5

It might be out of scope for this PR but I think it is worth doing it soon.

Reply:
Please let's not mix src and tests code right now. This needs a proper discussion and evaluation of pros/cons.

The main issue with this is that the tests should not depend and be bound on the implementation. This can hurt when refactoring things and eg adding i18n.

If you bring bugs into the code it can bring bugs to the tests and it might be hard to discover them. For example we had a discussion about bringing VMWrappers into the tests, but decided against it for this exact reason.

The main hardship is that we are not commiting code with the tests at the same. Once we have the CI, such renaming issue would get picked up immediately and such code would not get merged in the first place. So there is less pressure for sharing the constants IMO.

Reply:
I know it would be easier for now to do it like you suggest. But will get harder once you start introducing i18n.
Also, it would start a precedence with sharing the code, which might be confusing for newcomers.

Anyway, we can discuss this in a meeting next week if you want.